kong/runloop/plugin_servers/mp_rpc.lua

```diff
@@ -325,10 +327,16 @@ end
 
 
 function Rpc:handle_event(plugin_name, conf, phase)
-  local instance_id = self.get_instance_id(plugin_name, conf)
-  local _, err = bridge_loop(self, instance_id, phase)
+  local instance_id, err = self.get_instance_id(plugin_name, conf)
+  local _
```

Comment:
We can define `local _, instance_id, err`, then use them to get a relatively clean code.